Modules/BiddingManager/GUI.lua

```diff
@@ -386,6 +392,7 @@ function BiddingManagerGUI:StartAuction(show, auctionInfo)
     if myProfile then
         local roster = RosterManager:GetRosterByUid(self.auctionInfo:RosterUid())
         if roster then
+            self.auctionType = roster:GetConfiguration("auctionType")
```

Comment:
If we won't have the roster then the `self.auctionType` use in GUI will fail (there is no assumption that we do have knowledge about the roster we are in)